src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -1397,6 +1403,28 @@ string IndexColumnList(IndexColumn[] columns, string method)
             return builder.ToString();
         }
 
+        string IndexColumnsToTsVector(CreateIndexOperation operation, IModel model, IndexColumn[] columns, string configName)
+        {
+            string GetColumnExpression(IndexColumn column)
+            {
+                var property = FindProperty(model, operation.Schema, operation.Table, operation.Name);
+                var delimitedColumnName = Dependencies.SqlGenerationHelper.DelimitIdentifier(column.Name);
+                return (property?.IsColumnNullable() == true)
```

Comment:
Also, if the property isn't found, we should probably error here instead of generating the expression.

Reply:
This seems outdated, according to this line
https://github.com/npgsql/efcore.pg/commit/c6b846052b8fedfb5ff384466c52945a6a6540c9#diff-7d6fd89c6ae4f223fc36e554f5663a2dR1184